internal/gencli/cmd_file.go

```diff
@@ -276,32 +276,15 @@ var {{$methodCmdVar}} = &cobra.Command{
 		{{ end }}
 		{{ end }}
 		{{ else if and .Paged (not .IsLRO ) }}
-		// get requested page
-		var items []interface{}
-		data := make(map[string]interface{})
-
-		// PageSize could be an integer with a specific precision.
-		// Doing standard i := 0; i < PageSize; i++ creates i as
-		// an int, creating a potential type mismatch. 
-		for i := {{ .InputMessageVar }}.PageSize; i > 0; i-- {
-			item, err := iter.Next()
-			if err == iterator.Done {
-				err = nil
-				break
-			} else if err != nil {
-				return err
-			}
-
-			items = append(items, item)
+		// populate iterator with a page
+		_, iterErr := iter.Next()
+		if iterErr != nil && iterErr != iterator.Done {
+			return iterErr
 		}
-
```

Comment:
Please keep the new line between the error check closing brace and the beginning if the `if Verbose {`

Reply:
Fixed